src/components/application_manager/src/app_launch/app_launch_data_db.cc

```diff
@@ -293,6 +293,7 @@ bool AppLaunchDataDB::Clear() {
   bool retVal = false;
 
   utils::dbms::SQLQuery query(db());
+  // cppcheck-suppress redundantAssignment
   retVal = query.Exec(kDropSchema);
```

Comment:
bool retVal = query.Exec(kDropSchema);

Reply:
Fixed in d4c6de32f5592edc35229f55afdec6c02c55b0e8
Please check